### ISSUE TITLE
Update: Homepage search bar

### ISSUE THREAD

**Issue by reynalddizon** [reynalddizon/MapStoryToDo#5]
Update: Homepage search bar.
Now that we have separated Content search from User search, the homepage search bar should just focused on Content search. Placeholder text must be updated as well.

**Comment by travislbrundage** [comment]
I would presume this refers to that search bar which says 'Explore events, places, periods and storytellers'?

So I'm guessing it should say instead something like 'Explore MapStories and StoryLayers'? Or perhaps 'Explore events, places, periods, mapstories...'?

My assumption is the query will just redirect them to the search page, querying for any content that has a title with the text they entered. Will this be functionally identical to the 'quick search' bar?

**Comment by reynalddizon** [comment]
Yes! The homepage search bar and the mini search box in the navbar should have the same functionality, i.e. content search.

**Comment by reynalddizon** [comment]
And as for the placeholder, you're correct. It should just be "Explore Storylayers and Mapstories"

**Comment by travislbrundage** [comment]
Cool, thanks Rey. Wrapping up parallax and then will get this done. Not sure if it's just my localhost but on my VM the search bar is not functioning at all, so might need to either implement the functionality or fix whatever is bugged (haven't checked the code yet)

**Comment by jonpmarino** [comment]
Great, guys!
